src/server/protocol.ts

```diff
@@ -679,8 +679,8 @@ namespace ts.server.protocol {
 
     export type OrganizeImportsScope = GetCombinedCodeFixScope;
 
-    export interface OrganizeImportsRequestArgs {
-        scope: OrganizeImportsScope;
+    export interface OrganizeImportsRequestArgs extends OrganizeImportsScope {

```

Comment:
I'm confused by this change.  What happened to the `scope` property?

Reply:
Oh, sorry, I was a little confused on these - I'll change this one to be non-breaking.